from context import errors

import unittest


class ErrorsTest(unittest.TestCase):
    """
    Test the exceptions
    """

    def test_NestedJSONError(self):
        """
        """
        pass

    def test_CouldNotParseJSONError(self):
        """
        """
        pass

    def test_InvalidDBName(self):
        """
        """
        pass
